pkg/reconciler/workload/resource/resource_reconcile.go

```diff
@@ -68,6 +68,22 @@ func (c *Controller) reconcileResource(ctx context.Context, lclusterName logical
 
 	annotationPatch, labelPatch := computePlacement(ns, obj)
 
+	// If the object DeletionTimestamp is set, we should set all locations deletion timestamps annotations to the same value.
+	if obj.GetDeletionTimestamp() != nil {
+		klog.V(2).Infof("Resource is being deleted; setting the deletion per locations timestamps for %s|%s/%s", lclusterName, obj.GetNamespace(), obj.GetName())
```

Comment:
V(3). Our rule of thumb: one V(2) line per reconcile.

Reply:
done